shared/protocol.ts

```diff
@@ -179,6 +180,7 @@ export interface DeleteAcceptedMessage {
 }
 
 export const enum CopyFormat {
+	Raw = "Raw",
```

Comment:
"Hex Octets" would be a better name for this. Raw implies we're tossing binary onto your clipboard, which we aren't 🙂

Reply:
"Hex Octets" looks good.
I just don't know how to describe this structure briefly, so I describe it as what you get is what you see. XD

Reply:
changed by [c3820c4](https://github.com/microsoft/vscode-hexeditor/pull/540/commits/c3820c4ec6a1cbe86dad5eee8b55189d728acdf2)